flipperdevices/flipperzero-firmware
# [FL-905] Add battery health state monitoring in power app

# What's new

- Read battery health state from gauge registers in bq27220 driver
- Add battery health getter in api-hal-power
- Add battery health update in power app

# Verification

- Compile and upload to F5
- Compile and upload to F4
- Observe Health state in power application gui
- Observe Health state with cli command power_test

# Checklist (do not modify)
